src/main/java/io/jenkins/plugins/audit/listeners/UserLogListener.java

```diff
@@ -80,6 +80,8 @@ protected void loggedIn(@Nonnull String username) {
         Login login = LogEventFactory.getEvent(Login.class);
 
         RequestContext.setIpAddress(Stapler.getCurrentRequest().getRemoteAddr());
+        RequestContext.setNodeName("master");
```

Comment:
I suppose these work for now, but note that we need to move this to the servlet filter created in [JENKINS-54637](https://issues.jenkins-ci.org/browse/JENKINS-54637). The requestId will be set properly by that (which means you can't just call `RequestContext.clear()` after logging sometime; this will be done at the end of the request in the servlet filter instead), and the node name can be set to `master` there by default as well (which means we might be able to remove the default initializers in RequestContext eventually).

Reply:
hmh.. will keep that in mind :)